Graded/Usage/Decidable/Assumptions.agda

```diff
@@ -49,7 +48,7 @@ record Assumptions : Set a where
 
     instance
       -- A dedicated nr function is assumed to exist.
-      ⦃ has-nr ⦄ : Dedicated-nr
+      ⦃ has-nr ⦄ : Nr-available
```

Comment:
Could decidability also be proved under something like the following assumption?
```agda
Nr-not-available-GLB ×
(∀ r p q → ∃ λ s → Greatest-lower-bound s (nrᵢ r p q))
```
And does the second part of this assumption hold for at least some of the example modalities? (I think you and I have already discussed something like this.)